server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/TokenService.java

```diff
@@ -111,7 +111,7 @@ public TokenService(ProjectManager projectManager, CommandExecutor executor) {
             if (exist) {
                 // TODO(hyangtack) Would send 409 conflict when the following PR is merged.
                 // https://github.com/line/armeria/pull/746
-                throw new HttpResponseException(HttpStatus.CONFLICT);
+                throw HttpResponseException.of(HttpStatus.CONFLICT);
```

Comment:
@huydx Would you change this to `HttpStatusException.of(HttpStatus.CONFLICT)` in order to reuse a cached exception object? :-)